src/common/components/nav/nav.bundle.js

```diff
@@ -7,11 +7,11 @@
   // angular.js to load itself into
   const angular = (window.angular = {});
 
-  require('./nav.component');
+  const nav = require('./nav.component');
```

Comment:
Thanks for questioning this. It needed a `.default` on the end of it. `require` returns an object with all the named exports, including `default` as a key. I guess I'm used to working with `import`.